### DISCUSSION TITLE
[Feature Request] vant没有table得解决办法！！！！！！！

### DISCUSSION THREAD

**Discussion by tianheng2017** [youzan/vant#12302] [General]
### What problem does this feature solve?

各位，看好了，给你们看看如何降维打击

如果要求不复杂，把layui的css引入进来，然后使用html的table就行了

![image](https://github.com/youzan/vant/assets/29515610/940e2f4c-92ff-4f73-a81f-9cdd520b078b)

对表格要求较高，可以引入layui-vue和vant混用，非常的nice

![image](https://github.com/youzan/vant/assets/29515610/adebcbd8-3779-4a39-9bfb-09f9b50c3ce8)

### What does the proposed API look like?

vant + layui-vue是我找到的互补的可以解决问题的完美方案

**Comment by memory1668** [comment]
vxe-table也可https://vxetable.cn/v3/#/table/api
